src/SmartComponents/Actions/ListActions.js

```diff
@@ -44,9 +46,12 @@ class ListActions extends Component {
         this.setPage = this.setPage.bind(this);
         this.setPerPage = this.setPerPage.bind(this);
         this.parseProductCode = this.parseProductCode.bind(this);
+        this.onSearch = this.onSearch.bind(this);
     }
 
     componentDidMount() {
+        document.getElementById('root').classList.add('actions__list');
```

Comment:
ok I can stomach this... if we removed the class in the `componentDidUnmount` otherwise 😱thats gonna be one ova classy root after some time